action.yaml

```diff
@@ -77,15 +77,14 @@ runs:
         fi
 
         echo Scanning Docker container ${CONTAINER_IMAGE} with Mondoo Client...
-        mondoo scan --detect-cicd docker://${CONTAINER_IMAGE} --config mondoo.json --score-threshold ${{ inputs.score_threshold }} --output ${{ inputs.output_format }} ${{ inputs.extra_args }}
+        mondoo scan docker image ${CONTAINER_IMAGE} --detect-cicd --config mondoo.json --score-threshold ${{ inputs.score_threshold }} --output ${{ inputs.output_format }} ${{ inputs.extra_args }}
```

Comment:
do we need `--detect-cicd`

Reply:
i think not. I kept it since it was already there. I will give it a try without the flag now

Reply:
It works without the flag as well 🥳

Reply:
Super cool. Thank you